static-analysis-plugin/plugin/src/test/groovy/com/novoda/staticanalysis/PenaltyExtensionTest.groovy

```diff
@@ -0,0 +1,88 @@
+package com.novoda.staticanalysis
+
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+
+import static com.novoda.test.PenaltyExtensionSubject.assertThat
+
+class PenaltyExtensionTest {
```

Comment:
would it make sense to cover cases where someone tries to configure negative values? 😄